### PR TITLE
[one-build] Add an option, "substitute_strided_slice_to_reshape"

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This adds a new option, `substitute_strided_slice_to_reshape`
into one-build.

issue #7232
draft #7235

Signed-off-by: Hyun Sik Yoon <hyunsik.yoon@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/how-to-use-one-commands.txt [hunk 1]
```diff
@@ -126,7 +126,7 @@ one-import-tf
 This will convert TensorFlow model (.pb) file to our circle model. You can also
 directly call this command. one-import-tf invokes tf2tfliteV2.py script that
 will internally use TensorFlow lite converter and then invoke tflite2circle
-converter to convert tflite model to circle model.
```
Comment (hyunsik-yoon) [inline compiler/one-cmds/how-to-use-one-commands.txt:129]: Sorry about this change. My editor keeps remove this training space while saving this file. Can I keep this in this PR?

File: compiler/one-cmds/how-to-use-one-commands.txt [hunk 2]
```diff
@@ -126,7 +126,7 @@ one-import-tf
 This will convert TensorFlow model (.pb) file to our circle model. You can also
 directly call this command. one-import-tf invokes tf2tfliteV2.py script that
 will internally use TensorFlow lite converter and then invoke tflite2circle
-converter to convert tflite model to circle model. 
+converter to convert tflite model to circle model.
```
Comment (seanshpark) [inline compiler/one-cmds/how-to-use-one-commands.txt:129]: maybe leave this as is as not related to the purpose.

File: compiler/one-cmds/utils.py [hunk 3]
```diff
@@ -184,10 +186,10 @@ def _make_circle2circle_cmd(args, driver_path, input_path, output_path):
             if type(getattr(args, opt[0])) is bool:
                 cmd.append('--' + opt[0])
             """
-            This condition check is for config file interface, usually would be 
-             SomeOption=True 
-            but user can write as follows while development 
-             SomeOption=False
```
Comment (hyunsik-yoon) [inline compiler/one-cmds/utils.py:190]: ditto 😢

File: compiler/one-cmds/utils.py [hunk 4]
```diff
@@ -184,10 +186,10 @@ def _make_circle2circle_cmd(args, driver_path, input_path, output_path):
...
                 cmd.append('--' + opt[0])
             """
-            This condition check is for config file interface, usually would be 
-             SomeOption=True 
-            but user can write as follows while development 
-             SomeOption=False 
+            This condition check is for config file interface, usually would be
+             SomeOption=True
+            but user can write as follows while development
+             SomeOption=False
```
Comment (seanshpark) [inline compiler/one-cmds/utils.py:192]: does this change come from format checker?

File: compiler/one-cmds/how-to-use-one-commands.txt (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/how-to-use-one-commands.txt:129]: If possible can you send another PR for this and below change for remove space?

File: compiler/one-cmds/utils.py (hunk 4 above)
Comment (hyunsik-yoon) [reply compiler/one-cmds/utils.py:192]: No. I ran `./nnas format` with previous PRs but these trailing spaces were not removed. 😸 

I created another PR to remove these trailing spaces. #7294

**Comment by hyunsik-yoon** [comment]
Rebased upon #7294.

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**